### ISSUE TITLE
All these templates not running. Can you let me know the solution?

### ISSUE THREAD

**Issue by kashif979** [projectdiscovery/nuclei#5863]
### Is there an existing issue for this?

- [x] I have searched the existing issues.

### Current Behavior

![Image](https://github.com/user-attachments/assets/6faf891f-9fbc-4201-b66b-581d6afa07b9)

### Expected Behavior

I run nuclei for specific template like : /home/kali/nuclei-templates/workflows/wordpress-workflow.yaml

Showing an error "Could not run nuclei: no templates provided for scan"

### Steps To Reproduce

1. nuclei  -d example.com -t /home/kali/nuclei-templates/workflows/wordpress-workflow.yaml
2.  Respond me:
3.  INF] Current nuclei version: v3.3.2 (outdated)
[INF] Current nuclei-templates version: v10.0.4 (latest)
[WRN] Scan results upload to cloud is disabled.
[INF] Targets loaded for current scan: 3
[INF] No results found. Better luck next time!
[FTL] Could not run nuclei: no templates provided for scan

### Relevant log output

```shell

```

### Environment

```markdown
- Nuclei:  3.3.2
```

**Comment by geeknik** [comment]
Not a bug.  I recommend reading https://docs.projectdiscovery.io/templates/introduction but if you can't sit still long enough, try `nuclei -h | grep workflow`. 👍🏻

**Comment by dogancanbakir** [comment]
Please refer to https://docs.projectdiscovery.io/templates/introduction and https://docs.projectdiscovery.io/tools/nuclei/running for more detailed information. 

tl;dr; https://docs.projectdiscovery.io/tools/nuclei/running#executing-template-workflows

Closing this. Feel free to reopen if you have any other questions.
